tldw_Server_API/app/api/v1/endpoints/writing_manuscripts.py

```
@@ -819,3 +844,1048 @@ async def delete_scene(
+
+@router.post(
+    "/projects/{project_id}/characters",
+    response_model=ManuscriptCharacterResponse,
+    status_code=status.HTTP_201_CREATED,
+    summary="Create a manuscript character",
+    tags=["manuscripts"],
+)
+async def create_character(
+    project_id: str,
+    payload: ManuscriptCharacterCreate,
+    db: CharactersRAGDB = Depends(get_chacha_db_for_user),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)